media-video/gxine/files/gxine-0.5.909-fno-common.patch

```diff
@@ -0,0 +1,21 @@
+--- a/src/main.c
```

Comment:
Bug reference?

Reply:
Oh, it's an upstream patch. Link too then!

Reply:
Done. This one's not upstream ;)